yellowbrick/regressor/residuals.py

```diff
@@ -422,7 +429,10 @@ def __init__(self, model, ax=None, hist=True, train_color='b',
         # Store labels and colors for the legend ordered by call
         self._labels, self._colors = [], []
 
-        self.alpha = alpha
+        self.alphas = {
+                'train_point': train_alpha,
+                'test_point':test_alpha
```

Comment:
tiny thing here, missing space between `'test_point':` and  `test_alpha`

Reply:
Done!